cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateControllerCommandOptions.java

```diff
@@ -7,7 +7,7 @@
  */
 public class CreateControllerCommandOptions {
 
-    @CommandLine.Option(names = {"-y", "--yaml"}, required = true)
+    @CommandLine.Option(names = {"-y", "--yaml"}, required = true, description = "The path to the yaml file.")
```

Comment:
You can import the CommandLine, so that you will just use the @Option.

Reply:
Refactored.